yjit/src/options.rs

```diff
@@ -100,8 +106,9 @@ pub static mut OPTIONS: Options = Options {
 };
 
 /// YJIT option descriptions for `ruby --help`.
-static YJIT_OPTIONS: [(&str, &str); 9] = [
-    ("--yjit-exec-mem-size=num",           "Size of executable memory block in MiB (default: 48)."),
+pub const YJIT_OPTIONS: &'static [(&str, &str)] = &[
+    ("--yjit-mem-size=num",                "Soft limit of all memory bytes for YJIT in MiB (default: 128)."),
```

Comment:
ditto with https://github.com/ruby/ruby/pull/11810/files#r1790464198

Reply:
Your description fails `test_usage_long`. Mine was carefully crafted to fit exactly in 80 characters (including indentation), so you can't make it longer.

Reply:
How about this? https://github.com/ruby/ruby/pull/11810/commits/288465deee2febbf46e2c200a851cc7b5ba3b0af